docs/design/sql/20-match-jet-job-to-sql.md

```diff
@@ -0,0 +1,92 @@
+# Matching the SQL query to the Jet job
+
+|||
+|---|---|
+|Related Jira|[HZ-2859](https://hazelcast.atlassian.net/browse/HZ-2859)|
+|Document Status / Completeness|IN PROGRESS|
+|Requirement owner|Saumitra Sahi|
+|Developer(s)|Sasha Syrotenko, Ivan Yaschishin|
+|Quality Engineer|Isaac Sumner|
+|Technical Reviewers|Krzystof Jamroz|
+|Version|5.4|
+
+## Background
+In Hazelcast Platform 5.3, the sole SQL job metrics report can be accessed exclusively through
+the `getJobAndSqlSummaryList()` method within the Java client. This approach doesn't align with
+the idea of SQL serving as the primary Hazelcast interface. Consequently, there is currently
```

Comment:
> idea of SQL serving as the primary Hazelcast interface

this TDD does not affect it. Query->Job mapping will not be in SQL, but in client API

Reply:
> this TDD does not affect it.  Query->Job mapping will not be in SQL, but in client API

Not really. I phrased it with the mind, that SQL, which is declared as a main entry to Hazelcast Platform, doesn't provide accessible metrics for  Jet jobs, launched by SQL.